serverless-plugin/tests/alarms-dynamodb.test.js

```diff
@@ -0,0 +1,120 @@
+'use strict'
+
+const dynamoDbAlarms = require('../alarms-dynamodb')
+const CloudFormationTemplate = require('../cf-template')
+
+const { test } = require('tap')
+const { cloneDeep } = require('lodash')
+
+const defaultConfig = require('../default-config')
+const {
+  assertCommonAlarmProperties,
+  alarmNameToType,
+  createTestConfig,
+  defaultCfTemplate
+} = require('./testing-utils')
+
+const sls = {
+  cli: {
+    log: () => {}
+  }
+}
+
+const context = {
+  topicArn: 'dummy-arn',
+  stackName: 'testStack',
+  region: 'eu-west-1'
+}
+
+const alarmConfig = createTestConfig(
+  defaultConfig.alarms, {
+    DynamoDB: {
+      Period: 900,
+      ReadThrottleEvents: {
+        Threshold: 10
+      },
+      WriteThrottleEvents: {
+        Threshold: 20
+      },
+      UserErrors: {
+        Threshold: 100
+      },
+      SystemErrors: {
+        Threshold: 200
+      }
+    }
+  })
+
+const dynamoDbAlarmConfig = alarmConfig.DynamoDB
+
+test('DynamoDB alarms are created', (t) => {
+  const { createDynamoDbAlarms } = dynamoDbAlarms(dynamoDbAlarmConfig, context)
+  const compiledTemplate = cloneDeep(require('./resources/cloudformation-template-stack.json'))
+  const cfTemplate = CloudFormationTemplate(compiledTemplate, {}, sls)
```

Comment:
```suggestion
  const cfTemplate = createTestCloudFormationTemplate()
```

I am not 100% sure about this one because I believe you are passing an extra parameter, but it would be good to test this one out or maybe making that helper function more generic.